make/autoconf/lib-vmath.m4

```diff
@@ -0,0 +1,125 @@
+#
+# Copyright (c) 2023, Arm Limited. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.  Oracle designates this
+# particular file as subject to the "Classpath" exception as provided
+# by Oracle in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+# or visit www.oracle.com if you need additional information or have any
+# questions.
+#
+
+###############################################################################
+#
+# Setup vmath framework and check its dependences
+#
+AC_DEFUN_ONCE([LIB_SETUP_VMATH],
+[
+  AC_ARG_WITH(libsleef, [AS_HELP_STRING([--with-libsleef],
+      [specify prefix directory for the libsleef library
+      (expecting the libraries under PATH/lib and the headers under PATH/include)])])
+
+  ENABLE_LIBSLEEF=false
+  LIBVMATH_CFLAGS=
+  LIBVMATH_LIBS=
+
+  if test "x$NEEDS_LIB_SLEEF" = xfalse; then
+    if test "x${with_libsleef}" != "x" &&
+       test "x${with_libsleef}" != "xno"; then
+      AC_MSG_WARN([[libsleef is not used, so --with-libsleef is ignored]])
+    fi
+  else
+    LIBSLEEF_FOUND=no
+
+    if test "x${with_libsleef}" = "xno"; then
+      AC_MSG_NOTICE([libvmath will not be built, because its dependence libsleef is disabled])
+    else
+      if test "x${with_libsleef}" != "x" &&
+         test "x${with_libsleef}" != "xyes"; then
+        # Check the specified libsleef
+        AC_MSG_CHECKING([for the specified LIBSLEEF])
+        if test -e ${with_libsleef}/lib/libsleef.so &&
+           test -e ${with_libsleef}/include/sleef.h; then
+          LIBSLEEF_FOUND=yes
+          LIBVMATH_LIBS="-L${with_libsleef}/lib"
+          LIBVMATH_CFLAGS="-I${with_libsleef}/include"
+          AC_MSG_RESULT([yes])
+        else
+          AC_MSG_RESULT([no])
+          AC_MSG_ERROR([Could not locate libsleef.so or sleef.h in ${with_libsleef}])
+        fi
+      else
+        # Check for the libsleef under system locations
+        # Do not try pkg-config if we have a sysroot set.
+        if test "x$SYSROOT" = "x" &&
+           test "x${LIBSLEEF_FOUND}" = "xno"; then
+          PKG_CHECK_MODULES([LIBSLEEF], [sleef], [LIBSLEEF_FOUND=yes], [LIBSLEEF_FOUND=no])
```

Comment:
```suggestion
          PKG_CHECK_MODULES([SLEEF], [sleef], [LIBSLEEF_FOUND=yes], [LIBSLEEF_FOUND=no])
```

Otherwise `PKG_CHECK_MODULES` will set the variables  LIBSLEEF_CFLAGS and LIBSLEEF_LIBS.

Reply:
Keep using `LIBSLEEF`, as the cflags and libs are named with `LIBSLEEF_` prefix. Thanks!